packages/serverless/package.json

```diff
@@ -0,0 +1,91 @@
+{
+  "name": "@sentry/serverless",
```

Comment:
I would like to reiterate. I do not believe this needs a new package - there is no use for a new client/backend. You can simply add the aws_lambda integration to @sentry/node.

You can make it so that the aws_context is an option they must pass into the constructor of the aws_lambda integration

Reply:
> You can make it so that the aws_context is an option they must pass into the constructor of the aws_lambda integration

This has been not changed, and it's related to my previous comment about `Options`.

> You can simply add the aws_lambda integration to @sentry/node.

It still lives in a separate package, despite everyone agreeing that it should be an in integration.